src/Illuminate/Validation/Rules/Password.php

```diff
@@ -123,7 +123,7 @@ public static function default()
     {
         $password = value(static::$defaultCallback);
 
-        return $password instanceof static ? $password : static::min(8);
+        return $password instanceof Rule ? $password : static::min(8);
```

Comment:
Surely `self` is the correct thing to put here?

Reply:
`Laravel\Fortify\Rules\Password` just implements `Illuminate\Contracts\Validation\Rule`. At current start you can set defaults by returning `Laravel\Fortify\Rules\Password`.

Reply:
Without this changes:

```php
use Laravel\Fortify\Rules\Password as FortifyPassword;
use Illuminate\Validation\Rules\Password;

Password::defaults(function () {
    return (new FortifyPassword())->length(10);
});

Password::defaults() // return Illuminate\Validation\Rules\Password with min 8 instead of 10.
```

Reply:
Ok. I think we probs wanna add a test covering this, to check this doesn't break in the future.